index.html

```diff
@@ -1,14 +1,168 @@
+          Integer euismod efficitur facilisis. Aliquam eget auctor ligula.
+
+        </p>
+        <a href="continue-reading.html"><button class="nav-buttons">CONTINUE READING</button></a>
+      </article>
+    </div>
+    <!--Section for blogs and posts displayed vertically on the side of the page-->
+    <div id="side-bar">
+      <section id="blog-posts">
+        <div class="blog-post">
+          <a href="blog-post.html"><button class="blog-post-button">DATE</button></a>
+          <h6>JUST ANOTHER POST</h6>
+          <p>
+            Aenean non est non purus eleifend maximus. Nunc a lectus at eros
+            aliquam mollis ac a justo.
+          </p>
+        </div>
+        <hr class="side-bar-hr">
+        <div class="blog-post">
+          <a href="blog-post.html"><button class="blog-post-button">DATE</button></a>
+          <h6>JUST ANOTHER POST</h6>
+          <p>
+            Aenean non est non purus eleifend maximus. Nunc a lectus at eros
+            aliquam mollis ac a justo.
+          </p>
+        </div>
+        <hr class="side-bar-hr">
+        <div class="blog-post">
+          <a href="blog-post.html"><button class="blog-post-button">DATE</button></a>
+          <h6>JUST ANOTHER POST</h6>
+          <p>
+            Aenean non est non purus eleifend maximus. Nunc a lectus at eros
+            aliquam mollis ac a justo.
+          </p>
+        </div>
+        <hr class="side-bar-hr">
+        <hr class="side-bar-hr">
+      </section>
+      <section id="note">
+        <div class="note">
+          <h5>SOMETHING OF NOTE</h5>
+          <img src="https://www.placecage.com/75/75" />
+          <p>Vestibulum iaculis purus sed purus dignissim, mollis pulvinar magna
+            tincidunt. Praesent tempus elementum euismod. Suspendisse vitae mauris
+            ante. </p>
+          <a href="learn-more.html"><button class="nav-buttons">LEARN MORE</button></a>
+
+        </div>
+        <hr class="side-bar-hr">
+        <div class="note">
+          <h5>SOMETHING OF LESS NOTE</h5>
+          <img src="https://www.placecage.com/75/75" />
+          <p>Vestibulum iaculis purus sed purus dignissim, mollis pulvinar magna
+            tincidunt. Praesent tempus elementum euismod. Suspendisse vitae mauris
+            ante. </p>
+          <a href="learn-more.html"><button class="nav-buttons">LEARN MORE</button></a>
+
+        </div>
+
+      </section>
+    </div>
+  </section id="main">
+<!--Section for footer, containing the form element-->
+  <section id="footer">
+    <h4>QUESTIONS OR COMMENTS? GET IN TOUCH:</h4>
+    <div class="grid-container">
+
+      <input class="grid-item-name" type="text" name="name" placeholder="Name" />
+      <input class="grid-item-email" type="text" name="email" placeholder="Email" />
+      <textarea class="grid-item-note" type="text" name="note" placeholder="Note"></textarea>
+
+    </div>
+    <button id="send-message">SEND MESSAGE</button>
+
+
+
+
+
+  </section>

```

Comment:
This section is a good start to the footer example in the remock screen shots. However, you will need to add in the [additional social links and icons to this section](https://files.slack.com/files-pri/TAR9B9W87-F01MT5DSD6E/home_page.jpg) to wrap up the minimum required page layout/content.